autoref/ref.py

```diff
@@ -1,105 +1,52 @@
-import os
-from time import mktime
 from datetime import datetime
 import ROOT
-import json
 
-import sql
+def get_ref_cands(refs):
```

Comment:
Could you add a comment describing the format and contents of what's returned by `get_ref_cands`?